lib/ansible/modules/inventory/group_by.py

```diff
@@ -33,6 +33,12 @@
     description:
     - The variables whose values will be used as groups
     required: true
+  parent:
```

Comment:
I thought it could be solved by loop (e.g. with_items), but if you decide that 'parents' option is a better idea, i can implement it.